mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -77,6 +77,8 @@ CLASS cub net/minecraft/client/texture/NativeImage
 		ARG 7 shiftX
 		ARG 8 shiftY
 		ARG 10 glyph
+	METHOD a readPixels (Z)V
```

Comment:
Can we come up with a better name for this?

Reply:
It literally reads the pixel data from the GlStateManager. `readPixelData` is my other idea.

Reply:
So it captures pixel data from the screen and sets it in the `NativeImage`?

Reply:
Yes.

Reply:
Why not captureDisplay, or something akin?